frontend/src/index.tsx

```diff
@@ -1,69 +1,14 @@
+import './index.css';
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';
 import App from './App';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const container = document.getElementById('root');
+const root = createRoot(container!);
+root.render(<App />);
 
-async function loadAxe() {
```

Comment:
This removal could be done in a separate commit/PR.

Reply:
it's now in a separate commit